libs/core/langchain_core/beta/runnables/context.py

```diff
@@ -307,7 +307,34 @@ async def ainvoke(
 
 
 class Context:
-    """Context for a runnable."""
+    """
+    Context for a runnable.
+
+    The `Context` class provides methods for creating context scopes,
+    getters, and setters within a runnable. It allows for managing
+    and accessing contextual information throughout the execution
+    of a program.
+
+    Example:
+        .. code-block:: python
+
+            from langchain_core.beta.runnables.context import Context
+
```

Comment:
@aaronjimv  --  Thanks for the doc-string!

Would you mind adding an example of it being used inside of an LCEL chain -- Context is not really meant to be used outside off a chain

Reply:
Hi @eyurtsev thanks for the feedback. I still new using langchain so let me know any comment.